src/modules/microdds_client/dds_topics.yaml

```diff
@@ -7,7 +7,127 @@ publications:
 
   - topic: /fmu/out/collision_constraints
     type: px4_msgs::msg::CollisionConstraints
-
+    
+  - topic: /fmu/out/actuator_outputs
+    type: px4_msgs::msg::ActuatorOutputs
+
+  - topic: /fmu/out/actuator_servos
+    type: px4_msgs::msg::ActuatorServos
+   
+  - topic: /fmu/out/actuator_servos_trim
+    type: px4_msgs::msg::ActuatorServosTrim
+    
+  - topic: /fmu/out/airspeed
+    type: px4_msgs::msg::Airspeed
+    
+  - topic: /fmu/out/cellular_status
+    type: px4_msgs::msg::CellularStatus
+      
+  - topic: /fmu/out/collision_report
+    type: px4_msgs::msg::CollisionReport
+      
+  - topic: /fmu/out/differential_pressure
+    type: px4_msgs::msg::DifferentialPressure
+    
+  - topic: /fmu/out/distance_sensor
+    type: px4_msgs::msg::DistanceSensor
+    
+  - topic: /fmu/out/failure_detector_status
+    type: px4_msgs::msg::FailureDetectorStatus
+    
+  - topic: /fmu/out/follow_target
+    type: px4_msgs::msg::FollowTarget
+
+  - topic: /fmu/out/follow_target_estimator
+    type: px4_msgs::msg::FollowTargetEstimator
+    
+  - topic: /fmu/out/follow_target_status
+    type: px4_msgs::msg::FollowTargetStatus
+    
+  - topic: /fmu/out/position_controller_status
+    type: px4_msgs::msg::PositionControllerStatus
+    
+  - topic: /fmu/out/position_controller_landing_status
+    type: px4_msgs::msg::PositionControllerLandingStatus
+    
+  - topic: /fmu/out/position_setpoint
+    type: px4_msgs::msg::PositionSetpoint
+  
+  - topic: /fmu/out/sensor_accel
+    type: px4_msgs::msg::SensorAccel
+    
+  - topic: /fmu/out/sensor_baro
+    type: px4_msgs::msg::SensorBaro
+    
+  - topic: /fmu/out/sensor_gps
+    type: px4_msgs::msg::SensorGps
+    
+  - topic: /fmu/out/sensor_gyro
+    type: px4_msgs::msg::SensorGyro
+
+  - topic: /fmu/out/sensor_hygrometer
+    type: px4_msgs::msg::SensorHygrometer
+  
+  - topic: /fmu/out/wind
+    type: px4_msgs::msg::Wind  
+    
+  - topic: /fmu/out/sensor_mag
+    type: px4_msgs::msg::SensorMag
```

Comment:
why do you want the raw magnetometer data accessible?

Reply:
Hello, for me and perhaps for other people yes... If i want to make a kalman in my raspbery pi with ros2 to fusion with others imu for exemple.

Reply:
Right, but if someone wanted to do that, they can always do it. Here you are modifying the default, therefore I don't think it is necessary to make this default